projects/cryptofuzz/Dockerfile

```diff
@@ -17,10 +17,6 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 
 RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates gnupg software-properties-common wget && \
-    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add - && \
-    apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main' && \
```

Comment:
cc @guidovranken: I gave up here, so I'll let you take this over, if you want.

Reply:
Thanks @MarcoFalke !